.changeset/curvy-houses-grin.md

```diff
@@ -0,0 +1,5 @@
+---
+'@directus/app': patch
+---
+
+Fixed the failing auto-creating of default languages for custom languages collections in the translation field setup
```

Comment:
```suggestion
Fixed an issue where the creation of translation fields would fail when using an existing, custom language collection
```

Reply:
Funnily enough, at least in my tests, it didn't fail. But yeah, maybe something along those lines... 🤔